
The usual conclifts catched by gcc >= 3.4.x.

  - Rene Rebe <rene@exactcode.de>

--- mono-1.1.3/mono/mini/exceptions-ppc.c.vanilla	2004-12-07 23:33:57.000000000 +0100
+++ mono-1.1.3/mono/mini/exceptions-ppc.c	2005-02-04 10:49:59.579882290 +0100
@@ -226,6 +226,9 @@
 	return start;
 }
 
+static gpointer
+mono_arch_get_throw_exception_generic (guint8 *start, int size, int by_name, gboolean rethrow);
+
 /*
  * arch_get_call_filter:
  *
